fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -150,6 +151,44 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y, event=_ALL, **kwargs)
 
 
+class TruePositiveRateDifference(ConditionalSelectionRate):
+    r"""Implementation of Equal Opportunity as a moment.
+
+    Adds conditioning on label `y=1` compared to Demographic parity, i.e.
+
+    .. math::
+       P[h(X) = 1 | A = a, Y = 1] = P[h(X) = 1 | Y = 1] \; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines
+    the event corresponding to `y=1`.
+
+    The `prob_event` :class:`pandas:pandas.DataFrame` will record the
+    fraction of the samples corresponding to `y = 1` in
```

Comment:
nit: can you use the full line length? The way lines end seems a bit arbitrary here. We've had some people prefer lengths of 79 chars, others 99 (which is the max as defined in our repo), and I don't really care which one, but try to stick to one if possible :-)

Reply:
Sure! Sorry, I think I will also set my editor with these values so that I do not miss it. 👍